spec/models/spree/line_item_spec.rb

```diff
@@ -40,6 +52,10 @@ module Spree
           LineItem.without_tax.should == [li2]
         end
       end
+
+      it "finds line items sorted by name and unit_value" do
+        o.line_items.sorted_by_name_and_unit_value.should == [li5,li6,li4,li3]
```

Comment:
Could you please adopt the more recent RSpec syntax using `expect()` instead of `should()`?

`expect(o.line_items.sorted_by_name_and_unit_value).to eq([li5,li6,li4,li3])` or something like that